src/plugins/media-slideshow/media-slideshow.yml

```diff
@@ -2,6 +2,9 @@ mediaSlideshow:
   name:
     en: Media slideshow
     de: Medien-Slideshow
+  segments:
+    en: Segments
+    de: Segmente (oder Abschnitte?)
```

Comment:
It would definitely be streamlined, when the user see the "Timeline", it's always composed of the same things, "chapters". I thought we called them "segments" somewhere, but I will check and rename them.

Reply:
As discussed, let's call them the same everywhere :+1:

Reply:
They shall answer to the name of "Segment"/"Abschnitt" when called, from now on.